libs/shared/src/packages/utils-domain/names.ts

```diff
@@ -1,7 +1,7 @@
 export const domainNames = {
   // General
   area: 'Area',
-  location: 'Area',
+  location: 'Tag Location',
```

Comment:
We need to clairfy this one with PO. Previously it was decided that area and location was the same thing. It was decided that Area was the one we should use.

Reply:
I assumed that this ment it should be renamed:
![image](https://github.com/equinor/cc-components/assets/42774600/6ace1b2d-e4dc-4d80-b083-bd5adec06d74)

Reply:
@atlewee Should we stick with area for all apps to avoid confusion or is location more appropriate here?